notebooks/file_calibration.ipynb

```diff
@@ -574,10 +574,16 @@
    },
    "outputs": [],
    "source": [
-    "def array_class_plot(cls):\n",
-    "    fig, axes = plt.subplots(1, 2, figsize=(14, 6), dpi=100, gridspec_kw={'width_ratios': [2, 1]})\n",
-    "    plot_antclass(hd.antpos, cls, ax=axes[0], ants=[ant for ant in hd.data_ants if ant < 320], legend=False, title='HERA Core')\n",
-    "    plot_antclass(hd.antpos, cls, ax=axes[1], ants=[ant for ant in hd.data_ants if ant >= 320], radius=50, title='Outriggers')"
+    "def array_class_plot(cls, extra_label=\"\"):\n",
+    "    outriggers = [ant for ant in hd.data_ants if ant >= 320]\n",
+    "\n",
+    "    if outriggers:\n",
```

Comment:
FWIW, I prefer something more explicit like `if len(outriggers) == 0:` because I don't love having to remember that an empty list is False when converted to a boolean. Don't feel too strongly...

Reply:
Fair enough, changed that